src/FamilyHubs.ServiceDirectory.Shared/Dto/ServiceDeliveryDto.cs

```diff
@@ -6,5 +6,5 @@ namespace FamilyHubs.ServiceDirectory.Shared.Dto;
 public record ServiceDeliveryDto : ServiceDtoBase
 {
     //todo: why is the delivery type property called Name?
```

Comment:
//todo: why is the delivery type property called Name?
For 'Type' and 'Category' entities there's usually only a couple of attributes on the entity, typically 'Name' and 'Description'. So for DeliveryType or AttendingType etc you just have a 'Name' for the type of delivery or attending type. It's a mystery to me why they chose 'AttendingType', in my view 'ServiceType' would have been more appropriate. e.g the type of service is 'Telephone', 'Online',' In-Person'. In a conversation you would probably say it's a Telephone service or Online service or In-Person service.